sessions/3_visualization.md

```diff
@@ -4,4 +4,82 @@ title: "Hands-on session 3: Designing accessible biomedical data visualization"
 permalink: /visualization/
 ---
 
-This page is under construction.
+## Overview
```

Comment:
Good page! Perhaps we can make it a bit more structured by using a combination of h2 and h3? E.g., I see a few guidelines first (perhaps ## Guidelines then ### Chartability, ### Checlist, ### Color), etc. ? And I see a few regarding linear/circular, those can also be grouped together with one ## Layouts (for example) and then ### other headings)

Reply:
Ah, great point!